assets/alertmanager/alertmanager.yaml

```diff
@@ -149,7 +158,7 @@ spec:
       app.kubernetes.io/part-of: openshift-monitoring
       app.kubernetes.io/version: 0.22.2
   priorityClassName: system-cluster-critical
-  replicas: 3
+  replicas: 2
```

Comment:
Is this intended?

Reply:
yes I'll add more explanation to the commit message

Reply:
I've copied https://github.com/openshift/cluster-monitoring-operator/pull/1489/commits/0a05fb0daffd93ba743d15eca90be8262de694bf commit message into the PR description. Let me know if this is clear enough :)